cluster-up/cluster/kind/common.sh

```diff
@@ -39,7 +39,13 @@ ETCD_IN_MEMORY_DATA_DIR="/tmp/kind-cluster-etcd"
 
 function _wait_kind_up {
     echo "Waiting for kind to be ready ..."
-    while [ -z "$(docker exec --privileged ${CLUSTER_NAME}-control-plane kubectl --kubeconfig=/etc/kubernetes/admin.conf get nodes --selector=node-role.kubernetes.io/master -o=jsonpath='{.items..status.conditions[-1:].status}' | grep True)" ]; do
+    # For providers with version > 1.19 we use control-plane node role, master otherwise
```

Comment:
Does this check need to exist in perpetuity, or will we be able to retire it at some future point?

Reply:
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirtci/709/check-up-kind-1.19-sriov/1469329208550363136

kind job indeed failed, changes might be related ?

Reply:
@stu-gott We have to keep this check as long as we use a k8s 1.19 based provider. After that it can be deleted.

Reply:
@oshoval the changes was made to make kind 1.19 job works again.But for some reason it seems does not work. I'm investigating on it. Thanks

Reply:
Found it. I pushed the changes. The check was wrong